wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
   .ve-ui-mwAceEditorWidget .ace_editor {
     border-color: var(--cm-6);
   }
+	/*Blyad's Fixes*/
+	.central-featured-lang :hover, .central-featured-lang :hover strong::before, .other-project-link:hover, .central-featured-lang :hover small::before, .central-featured-lang :hover bdi::before, .central-featured-lang :hover span::before {
+		background-color: #0000;
```

Comment:
use existing css vars

also creating far too many rules, we already have rules enough for most cases. Use those.

Reply:
You dont have transparent rule, I could use `transparent` value, but this also works.

Reply:
We do.

one here

https://github.com/StylishThemes/Wikipedia-Dark/blob/ef5f2266d594e9c63612cf76ccae3306582d2fb4/wikipedia-dark.user.css#L439-L442

and above that there is one with !important on it.